save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/TestExecutionController.kt

```diff
@@ -93,7 +93,7 @@ class TestExecutionController(private val testExecutionService: TestExecutionSer
      * @param authentication
      * @return a list of [TestExecutionDto]s
      */
-    @GetMapping(path = ["/api/$v1/testLatestExecutions"])
+    @GetMapping(path = ["/api/$v1/testExecutions"])
```

Comment:
Why did you change `testLatestExecutions` to `testExecutions`

Reply:
Renamed